Taarafo.Core/Services/Foundations/PostImpressions/PostImpressionService.cs

```diff
@@ -38,5 +39,18 @@ public partial class PostImpressionService : IPostImpressionService
 
         public IQueryable<PostImpression> RetrieveAllPostImpressions() =>
            TryCatch(() => this.storageBroker.SelectAllPostImpressions());
+
+        public ValueTask<PostImpression> RemovePostImpressionByIdAsync(Guid postId, Guid profileId) =>
```

Comment:
Change to 
`public ValueTask<PostImpression> RemovePostImpressionAsync(PostImpression postImpression) =>`